benchexec/tools/verioover.py

```diff
@@ -0,0 +1,47 @@
+
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    VeriOover
+    """
+
+    def name(self):
+        return "VeriOover"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("VeriOover")
+    
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def cmdline(self, executable, options, task, rlimits):
+        if "-file" not in options:
+            options = options + ["-file"]
+
+        options = options + list(task.input_files_or_identifier)
```

Comment:
Is `-file` the parameter that the tool uses to receive the input file? If yes, then there should simply be no `-file` in options. There is no need for checking it here because it could be in a wrong position inside `options` anyway. Just add it unconditionally like with `-spec`.

Furthermore, does your tool support tasks that consist of more than one input file, or have no input file at all? If not, please do not use `task.input_files_or_identifier` but one of the other properties of `task` such that users get a nice error message.